examples/granger_causality.py

```diff
@@ -387,7 +377,7 @@
 try:
     spectral_connectivity_epochs(
         epochs, method=['gc'], indices=indices_ab, fmin=5, fmax=30, rank=None,
-        gc_n_lags=20)  # A => B
+        gc_n_lags=20, verbose=False)  # A => B
     print('Success!')
 except RuntimeError as error:
     print('\nCaught the following error:\n' + repr(error))
```

Comment:
If you want, you could add this to `expected_failing_examples` then you don't have to try/except at all here. Just let it actually fail and sphinx-gallery will print a nicely formatted traceback for you.

https://sphinx-gallery.github.io/stable/configuration.html#dont-fail-exit

Reply:
The traceback in the file works really nicely, but is it possible to overwrite the "BROKEN" thumbnail?
![image](https://github.com/mne-tools/mne-connectivity/assets/56922019/28820095-9029-4495-b2ac-c51c5246d0c3)
![image](https://github.com/mne-tools/mne-connectivity/assets/56922019/d6680b8d-78f7-4b40-a3d5-664427876f30)

E.g. setting `# sphinx_gallery_thumbnail_path = '_static/granger_causality_gallery_thumbnail.png'` within the example did not have an effect.

Reply:
Hmm not at the moment. Feel free to open an issue at https://github.com/sphinx-gallery/sphinx-gallery about adding this possibility

Reply:
Opened! https://github.com/sphinx-gallery/sphinx-gallery/issues/1220

If the others agree, I would stick with the try-except approach until this behaviour in sphinx gallery is changed, so as not to give the impression from the thumbnail that the entire example is about failing code.

Reply:
I think this is fine as is already, no need to wait to merge for this. It's an easy follow up PR after this is merged and SG has the machinery it needs